samples/web/content/apprtc/js/storage.js

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+/* More information about these options at jshint.com/docs/options */
+
+/* exported Storage */
+/* globals isChromeApp, chrome */
+
+'use strict';
+
+var Storage = function() {};
+
+// Get a value from local browser storage. Calls callback with value.
+// Handles variation in API between localStorage and Chrome app storage.
+Storage.prototype.getStorage = function(key, callback) {
+  if (isChromeApp()) {
+    // use chrome.storage.local
+    chrome.storage.local.get(key, function(values) {
+      // unwrap key/value pair
+      if (callback)
+      {
+        window.setTimeout(function() {
```

Comment:
@juberti suggested changing it from a direct call to callback to avoid possible reentrancy issues.